lib/features/sessions/cubit/bookmark_session_cubit.dart

```diff
@@ -34,6 +36,32 @@ class BookmarkSessionCubit extends Cubit<BookmarkSessionState> {
         bookmarkStatus: bookmarkStatus == BookmarkStatus.bookmarked,
       );
 
+      if (bookmarkStatus == BookmarkStatus.bookmarked) {
+        final session = await _dBRepository.getSession(sessionId);
+        final endTime = session?.endDateTime;
+        final notifcationTime = endTime?.subtract(const Duration(minutes: 5));
+
+        await AwesomeNotifications().createNotification(
+          content: NotificationContent(
+            id: sessionId,
+            channelKey: 'session_channel',
+            title: '${session?.title} feedback',
+            body:
+                'Please provide feedback for the session you ${session?.title} '
+                'just attended',
+          ),
+          schedule: NotificationCalendar.fromDate(date: notifcationTime!),
+        );
+
+        await NotificationService().showScheduledNotification(
+          id: sessionId,
+          channelKey: 'session_channel',
+          title: '${session?.title} feedback',
+          body: 'Please provide feedback for the session you ${session?.title} '
+              'just attended',
+          notificationTime: notifcationTime,
+        );
+      }
```

Comment:
We can make this neater by moving all of this into `notification_service`.  You can make a specialised method to take in the session. 

Then to use that method, you can inject this class into this cubit, the same way `_dBRepository` is injected once you have registered it as a singleton.